### PR TITLE
Updated ember.

### PR DESCRIPTION
**Pull request by craigsketchley**
I found this was breaking my app tests so updating ember and removing bower did the trick for me.

### CODE REVIEW

**Comment by jrowlingson** [comment]
@craigsketchley 👋  thanks for this - we will want update the [blueprint](https://github.com/craigsketchley/ember-cli-file-picker/blob/master/blueprints/ember-cli-file-picker/index.js#L7) to remove bower as well.

**Comment by craigsketchley** [comment]
Cheers. Ah, I missed the blueprint. I'll fix that up.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-cli-file-picker",
   "description": "An addon for ember-cli that provides a component to easily add a filepicker to your ember-cli app.",
-  "version": "0.0.10",
+  "version": "0.1.0",
```
Comment (craigsketchley) [inline package.json]: I wasn't sure how you'd want to increment the version number, so happy to change this if needed.

**Comment by jrowlingson** [comment]
@fsmanuel any feedback on this?

File: package.json (hunk 1 above)
Comment (fsmanuel) [inline package.json]: @craigsketchley can you please undo it. I use ember-cli-release to manage the version

Comment (fsmanuel) [inline package.json]: @craigsketchley I'll merge and fix it!

**Comment by fsmanuel** [comment]
@craigsketchley Thanks!

**Comment by fsmanuel** [comment]
@craigsketchley @jrowlingson released as `0.0.11`

**Pull request merged**